packages/dashboard/src/customization/propertiesSections/thresholdSettings/thresholdComponent.tsx

```diff
@@ -103,34 +104,46 @@ export const ThresholdComponent: FC<{
               className='threshold-configuration'
               style={{ gap: awsui.spaceScaledS }}
             >
-              <Box variant='span'>{defaultMessages.if}</Box>
-              <Select
-                expandToViewport={true}
-                options={comparisonOptions}
-                selectedOption={selectedOption}
-                onChange={onUpdateComparator}
-                data-test-id='threshold-component-operator-select'
-              />
-              <Input
-                value={`${displayValue}`}
-                placeholder='Threshold value'
-                onChange={onUpdateThresholdValue}
-                data-test-id='threshold-component-value-input'
-                invalid={!validValue}
-              />
-              {defaultMessages.showAs}
-              <ColorPicker
-                color={color || DEFAULT_THRESHOLD_COLOR}
-                updateColor={onUpdateColor}
-                data-test-id='threshold-component-color-picker'
-              />
-              <Button
-                ariaLabel='delete threshold'
-                iconName='remove'
-                variant='icon'
-                onClick={onDelete}
-                data-test-id='threshold-component-delete-button'
-              />
+              <Box variant='span' margin={{ top: 'l' }}>
+                {defaultMessages.if}
+              </Box>
+              <FormField label='Operator'>
+                <Select
+                  expandToViewport
+                  options={comparisonOptions}
+                  selectedOption={selectedOption}
+                  onChange={onUpdateComparator}
+                  data-test-id='threshold-component-operator-select'
+                />
+              </FormField>
+              <FormField label='Value'>
+                <Input
+                  value={`${displayValue}`}
+                  placeholder='Threshold value'
+                  onChange={onUpdateThresholdValue}
+                  data-test-id='threshold-component-value-input'
+                  invalid={!validValue}
+                />
+              </FormField>
+              <Box variant='span' margin={{ top: 'l' }}>
+                {defaultMessages.showAs}
+              </Box>
+              <Box variant='span' margin={{ top: 'l' }}>
+                <ColorPicker
+                  color={color || DEFAULT_THRESHOLD_COLOR}
+                  updateColor={onUpdateColor}
+                  data-test-id='threshold-component-color-picker'
+                />
+              </Box>
+              <Box variant='span' margin={{ top: 'l' }}>
+                <Button
+                  ariaLabel='delete threshold'
+                  iconName='remove'
+                  variant='icon'
+                  onClick={onDelete}
+                  data-test-id='threshold-component-delete-button'
+                />
+              </Box>
             </div>
```

Comment:
There's four separate top margin's here. Is it possible to set the margin once? Maybe on a container element?

Reply:
`<SpaceBetween />` has an alignItems prop. Could that be used to center the elements instead?

Reply:
It doesn't work for `<SpaceBetween />`
![Screenshot from 2024-02-12 12-39-09](https://github.com/awslabs/iot-app-kit/assets/154328164/378223d1-e53a-4ac9-8417-76014131c4d5)

I had tried to adjust the alignment using `alignItems: center` at it's parent `<div className='threshold-display-summary'>`, but it doesn't change the alignment because it is already centered.
If I apply `alignItems: end`, the text appears at the bottom, in that case as well need to apply margin somewhere.
![image](https://github.com/awslabs/iot-app-kit/assets/154328164/d1dcd2ab-892c-44d8-937e-420d8050708e)

I had to apply margin individually to those 4 elements to make text appear in same line.